code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -777,6 +777,10 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		pai = C
 		user << "<span class='notice'>You slot \the [C] into [src].</span>"
 		updateUsrDialog()
+<<<<<<< HEAD
+	else if(is_type_in_list(C, contained_item)) //Checks if there is a pen
+		if(inserted_item)
+=======

```

Comment:
Yeah, tortoise git said I was having a merge conflict, I clicked "resolve" and I did a bunch of really stupid things after that.